README.md

```diff
@@ -4,18 +4,18 @@ Transform FHIR to dataset for ML applications
 
 ## FHIR2Dataset in Detail
 
-This project is still under development.
```

Comment:
🚀

Reply:
There was a typo, the sentence was twice Ahah!
Now it just remains: 
```
_FHIR2Dataset is still under active development!_
```